src/app/frontend/common/components/breadcrumbs/component.ts

```diff
@@ -101,7 +101,7 @@ export class BreadcrumbsComponent implements OnInit {
 
         // Explore the route tree to the root route (parent references have to be defined by us on
         // each route).
-        if (route && route.data && route.data.parent) {
+        if (route.data && route.data.parent) {
```

Comment:
I think we could simplify this even further and simply go with `route = route?.data?.parent`. An undefined route will break the loop anyway so `else` is not really needed.